bindings/kafka/metadata.yaml

```diff
@@ -0,0 +1,238 @@
+    type: string
+    required: true
+    description: |
+      A comma-separated list of Kafka brokers.
+    example: '"localhost:9092,dapr-kafka.myapp.svc.cluster.local:9093"'
+    binding:
+      output: true
+      input: true
+  - name: publishTopic
+    type: string
+    required: true
+    description: |
+      The topic to publish to.
+    example: '"mytopic"'
+    binding:
+      output: true
+  - name: consumerGroup
+    type: string
+    description: |
+      A kafka consumer group to listen on. Each record published
+      to a topic is delivered to one consumer within each consumer
+      group subscribed to the topic.
+    example: '"group1"'
+    binding:
+      input: true
+  - name: clientID
+    type: string
+    description: |
+      A user-provided string sent with every request to 
+      the Kafka brokers for logging, debugging, and auditing purposes.
+    example: '"my-dapr-app"'
+    binding:
+      output: true
+      input: true
+  - name: authRequired
+    type: bool
+    description: |
+     (Deprecated) Enable SASL authentication with the Kafka brokers.
+    example: '"false"'
+    binding:
+      output: true
+      input: true
+  - name: authType
+    type: string
+    required: true
+    description: | 
+      Configure or disable authentication. 
+      Supported values: "none", "password", "mtls", or "oidc".
+    example: '"none"'
+    binding:
+      output: true
+      input: true
+  - name: saslUsername
+    type: string
+    description: |
+      The SASL username used for authentication. 
+      Only required if authType is set to "password".
+    example: '"myuser"'
+    binding:
+      output: true
+      input: true
+  - name: saslPassword
+    type: string
+    senstive: true
+    description: |
+      The SASL password used for authentication. 
+      Can be secretKeyRef to use a secret reference. 
+      Only required if authType is set to "password".
+    example: '"mypassword"'
+    binding:
+      output: true
+      input: true
+  - name: saslMechanism
+    type: string
+    description: |
+      The SASL Authentication Mechanism you wish to use. 
+      Only required if authType is set to "password".
+    example: '"SHA-512"'
+    default: '"PLAINTEXT"'
+    binding:
+      output: true
+      input: true
+  - name: initialOffset
+    type: string
+    description: |
+      The initial offset to use if no offset was previously committed.
+      Should be "newest" or "oldest".
+    example: '"oldest"'
+    default: '"newest", "oldest"'
+    binding:
+      input: true
+  - name: maxMessageBytes
+    type: number
+    description: |
+      The maximum size in bytes allowed for a single Kafka message.
+    example: '2048'
+    default: '1024'
+    binding:
+      output: true
+      input: true
+  - name: consumeRetryInterval
+    type: duration
+    description: |
+      The interval between retries when attempting to consume topics.
+      Treats numbers without suffix as milliseconds.
```

Comment:
Incorrect, numbers without suffixes are treated as seconds. Just remove this line since we should encourage the use of durations, with a proper suffix.

```suggestion
```